module-new/src/components/dateTimeInput/dateTimeInput.component.tsx

```diff
@@ -15,34 +15,37 @@ import {
   IArmstrongReactSelectOption,
   NullOrUndefined,
 } from '../../types';
-import { assertNever, stripNullOrUndefined } from '../../utils';
+import { assertNever, concat, stripNullOrUndefined } from '../../utils';
 import { Button } from '../button';
+import { useArmstrongConfig } from '../config';
 import { Input, ITextInputProps } from '../input';
+import { Label } from '../label';
 import { SingleSelect } from '../reactSelect/singleSelect';
+import { ValidationErrors } from '../validationErrors';
 import { formatDate, getMonths, getYears } from './dateTimeInput.utils';
 
 import 'react-datepicker/dist/react-datepicker.css';
 import './dateTimeInput.theme.css';
 
-export interface IDatePickerConfig
-  extends Omit<
-    ReactDatePickerProps,
-    'onChange' | 'value' | 'selectsRange' | 'selected' | 'startDate' | 'endDate' | 'disabled' | 'dateFormat' | 'locale'
-  > {
+export type IDatePickerConfig = Omit<
+  ReactDatePickerProps,
+  'onChange' | 'value' | 'selectsRange' | 'selected' | 'startDate' | 'endDate' | 'disabled' | 'dateFormat' | 'locale'
+>;
+
+export interface IDateTimeInputProps {
+  /** Props to spread onto the React Datepicker component (see: https://reactdatepicker.com) */
+  config?: IDatePickerConfig;
+
   /** The datetime format as a string (e.g. `dd/MM/yyyy HH:mm`) */
-  dateFormat?: string;
+  format?: string;
```

Comment:
should this maybe be `dateTimeFormat` for consistency with the DateInput or are they now both `format`?